llvm/lib/Transforms/Instrumentation/AddressSanitizer.cpp

```diff
@@ -1848,10 +2006,18 @@ void AddressSanitizer::instrumentUnusualSizeOrAlignment(
 
   Value *AddrLong = IRB.CreatePointerCast(Addr, IntptrTy);
   if (UseCalls) {
-    if (Exp == 0)
-      IRB.CreateCall(AsanMemoryAccessCallbackSized[IsWrite][0],
-                     {AddrLong, Size});
-    else
+    if (Exp == 0) {
+      if (TargetTriple.isSPIR()) {
+        SmallVector<Value *> Args;
```

Comment:
Same here, `SmallVector` can be specialized for initial number of elements

Reply:
Done